### PR TITLE
Hidden nodes are correctly hidden now.

### PR DESCRIPTION
**Pull request by PetzeltA**
One test updated to reflect the problem with initial load

### CODE REVIEW

File: jstreegrid.js [hunk 1]
```diff
@@ -456,11 +457,25 @@
 					findDataCell(this.uniq,removeNodes,this._gridSettings.gridcols).remove();
 				}
 			}, this))
-			.on("hide_node.jstree", $.proxy(function (e, data) {
-				// remove out own data cells
-        findDataCell(this.uniq, data.node.id, this._gridSettings.gridcols).remove();
-				// hide any children
-        this._hide_grid(data.node);
+      .on("hide_node.jstree", $.proxy(function (e, data) {
```
Comment (deitch) [inline jstreegrid.js]: Should this noe be `show_node`? We are _removing_ `"jstree-grid-hidden"` class?

File: jstreegrid.js [hunk 2]
```diff
@@ -456,11 +457,25 @@
...
+        // remove out own data cells
+        findDataCell(this.uniq, data.node.id, this._gridSettings.gridcols).removeClass("jstree-grid-hidden");
+        if (data.node.state.opened) {
```
Comment (deitch) [inline jstreegrid.js]: 1. Do we need to recurse down? What if it goes down multiple layers?
2. Do we need to look at children at all? Shouldn't we get events for the children?

**Comment by deitch** [comment]
@petzeltA, is this one fully subsumed in #232? Once we merge that one in, will this one be unnecessary?

**Comment by PetzeltA** [comment]
Yes #232 was branches from #231

File: jstreegrid.js (hunk 2 above)
Comment (PetzeltA) [inline jstreegrid.js]: We have to test this.
I think if a node is hidden or shown again, we'll have to recurse down to open child-nodes.

**Comment by deitch** [comment]
Ok, then I am closing this one out, as #232 is almost ready for merge.

**Pull request closed without merging**